hack/goreleaser.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2018 the Heptio Ark contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# $PUBLISH must explicitly be set to 'true' for goreleaser
+# to publish the release to GitHub.
+if [ "${PUBLISH:-}" != "true" ]; then
```

Comment:
I've gotten in the habit of using double brackets instead of single. Not a deal breaker though.